src/character.h

```diff
@@ -597,6 +598,8 @@ class Character : public Creature, public visitable<Character>
         // turn the character expired, if -1 it has not been set yet.
         int turn_died = -1;
 
+        mutable pathfinding_settings path_settings;
```

Comment:
Ok, caching is an entirely legitimate use for `mutable` but can you document this as otherwise the keyword is often a euphemism for `hack`